pkg/azureclients/vmssclient/interface.go

```diff
@@ -28,9 +28,9 @@ import (
 
 const (
 	// APIVersion is the API version for VMSS.
-	APIVersion = "2019-07-01"
+	APIVersion = "2020-12-01"
 	// AzureStackCloudAPIVersion is the API version for Azure Stack
-	AzureStackCloudAPIVersion = "2019-07-01"
```

Comment:
I am pretty sure this will break azure stack

Reply:
reverted this change.